### ISSUE TITLE
get_transcripts no longer working

### ISSUE THREAD

**Issue by Garrettlynch** [jdepoix/youtube-transcript-api#113]
I had been using this API without any issues but today I keep getting "The video is no longer available" error for videos that are available and have transcripts, code below, error output below that:

```
    # import the YouTube transcript api downloader
    from youtube_transcript_api import YouTubeTranscriptApi

    # get subtitles
    subtitles = YouTubeTranscriptApi.get_transcripts(["5Cdv932eoU0"])

    # to test - print the results
    print(subtitles)
```

```
$ python downloader1.py
Traceback (most recent call last):
  File "downloader1.py", line 72, in <module>
    subtitles = YouTubeTranscriptApi.get_transcripts(["1mf06IwdpVM"])
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/youtube_transcript_api/_api.py", line 102, in get_transcripts
    raise exception
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/youtube_transcript_api/_api.py", line 99, in get_transcripts
    data[video_id] = cls.get_transcript(video_id, languages, proxies, cookies)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/youtube_transcript_api/_api.py", line 128, in get_transcript
    return cls.list_transcripts(video_id, proxies, cookies).find_transcript(languages).fetch()
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/youtube_transcript_api/_api.py", line 70, in list_transcripts
    return TranscriptListFetcher(http_client).fetch(video_id)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/youtube_transcript_api/_transcripts.py", line 34, in fetch
    self._extract_captions_json(self._fetch_html(video_id), video_id)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/youtube_transcript_api/_transcripts.py", line 42, in _extract_captions_json
    raise VideoUnavailable(video_id)
youtube_transcript_api._errors.VideoUnavailable: 
Could not retrieve a transcript for the video https://www.youtube.com/watch?v=1mf06IwdpVM! This is most likely caused by:

The video is no longer available

If you are sure that the described cause is not responsible for this error and that a transcript should be retrievable, please create an issue at https://github.com/jdepoix/youtube-transcript-api/issues. Please add which version of youtube_transcript_api you are using and provide the information needed to replicate the error. Also make sure that there are no open issues which already describe your problem!
```

**Comment by jdepoix** [comment]
Hi @Garrettlynch, I just checked the video `1mf06IwdpVM` you posted about and it in fact does not have a transcript. Is this the same for other videos which actually have transcripts? Also, which version of this module are you using?

**Comment by Garrettlynch** [comment]
> Hi @Garrettlynch, I just checked the video `1mf06IwdpVM` you posted about and it in fact does not have a transcript. Is this the same for other videos which actually have transcripts? Also, which version of this module are you using?

Hi, I've tried multiple videos using both `get_transcript("5Cdv932eoU0")` and `get_transcripts(["5Cdv932eoU0"])` - always the same `VideoUnavailable` error today. As I said it was working without issue until today. Nothing changed in the code.

I was using youtube-transcript-api 0.3.1. I just updated to 0.4.1 and it seems to be working now. The video without a transcript threw back a `TranscriptsDisabled` error and a video with a transcript has worked. Odd that the version that had been working stopped working but at least it works now - thanks :)
